lib/browser/mqtt.ts

```diff
@@ -12,8 +12,8 @@
 * permissions and limitations under the License.
 */
 
-import { MqttClient as MqttClientInternal, IClientOptions, ISubscriptionGrant, IUnsubackPacket, IPublishPacket } from "mqtt";
-import { AsyncClient } from "async-mqtt";
+import { AsyncClient, IClientOptions, ISubscriptionGrant, IUnsubackPacket, IPublishPacket, IConnackPacket } from "async-mqtt";
```

Comment:
In a follow up, we can get rid of async-mqtt, import mqtt directly, and just do the same wrapping async-mqtt does, it's not special.